src/components/Tailwind/Buttons/ApproveButton.tsx

```diff
@@ -5,7 +5,8 @@ import SuccessIcon from 'assets/svg/success-icon.svg'
 export enum ApproveButtonState {
   Approved,
   NotApproved,
-  Approving
+  Approving,
+  Disabled
```

Comment:
@geonellov looks like this additional state is not recognized by `ApproveButton` component yet. If you look at line 20 below, the button gets disabled only when the state is `NotApproved`, but there is no condition/check for `Disabled` state yet.

Reply:
@schystz condition for Disabled added.

Reply:
@geonellov shouldn't the condition be `&&` instead of `||`?

Reply:
@geonellov 1 outstanding feedback above

Reply:
@schystz it should be `||` because the NotApproved and Disabled state will not happen at the same time.